app/views.py

```diff
@@ -145,3 +146,13 @@ def post(self, request):
         # a tutorial
                 return render(request, 'thankyou.html', {'title': 'Thanks!'})
         return render(request, 'thankyou.html', {'title': 'Thanks!'})
+
+def autocomplete(request):
+    query = request.GET.get('query')
+    tags = Tag.objects.filter(name__icontains=query) 
+    taglist = []
+    for t in tags:
+        taglist.append(t.name)
+    taglist_json = {"suggestions":taglist}
+    taglist_string = json.dumps(taglist_json)
+    return HttpResponse(taglist_string)
```

Comment:
Instead of using `json` module, you could also have used Django's `JsonResponse` to send back a JSON object for autocomplete.